src/helpers/index.js

```diff
@@ -0,0 +1,153 @@
+import isEmpty from 'lodash.isempty'
+
+export const setHttpToken = (token) => {
+    if (isEmpty(token)) {
+        window.axios.defaults.headers.common['Authorization'] = null
+    }
+
+    window.axios.defaults.headers.common['Authorization'] = 'Bearer ' + token
+}
+
+export const getErrorMessage = (originalError) => {
+	var notifyText = '';
```

Comment:
Don't use var, it's old :) you already using const in react component, so use it everywhere